.github/workflows/compatibility_tests.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
 
   unittests_check_compatibility:
```

Comment:
just a small thing but we can follow the same structure and name this `unittest_check_compatibility_1_0`